types/wordpress__admin/index.d.ts

```diff
@@ -0,0 +1,45 @@
+// Type definitions for @wordpress/admin 5.8
```

Comment:
Does this package exist? Where can I find the source code?

Reply:
It doesn't. It's not an npm package, it's the wordpress-core itself which uses the defined objects.

Reply:
See https://github.com/DefinitelyTyped/DefinitelyTyped/blob/a585d5fd2c4aaffae5c8e74e0763c7406689cdf1/types/activex-access/index.d.ts#L1 for an example of how to write this header, and remove the tslint npm-naming exception. (Ideally, remove all the tslint exceptions. Given that this is a new package, there should be no reason to start out with a bunch of rules disabled.)

Reply:
Implemented changes per request!